tutorials/01-pymapdl.py

```diff
@@ -0,0 +1,255 @@
+"""
+.. _ref_pymapdl_tutorial:
+
+================
+PyMAPDL Tutorial
+================
+
+
+In this tutorial, we will introduce some of the capabilities of
+PyMAPDL applied to Aerospace students and advocates.
+
+"""
+
+#########################################################
+# Python + Ansys = PyAnsys
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# Pyansys is Ansys effort on making their products accessible
+# using Python programming language.
+#
+# Today we will focus on PyMAPDL with is the Python library
+# which exposes MAPDL (Ansys Structural)
+#
+
+##########################################################
+# PyMAPDL
+# ~~~~~~~
+# MAPDL provides many tools and features which would require
+# A LOT of time to explain, so let's have a quick overview
+
+##########################################################
+# Launching PyMAPDL
+# -----------------
+# But first, let's launch PyMAPDL.
+from ansys.mapdl.core import launch_mapdl
+
+mapdl = launch_mapdl()
+
+print(mapdl)
+
+############################################################
+# Geometry
+# --------
+#
+# PyMAPDL support points (keypoints), lines, areas, and volumes
+# for geometry definition.
+#
+# You can plot an area using keypoints:
+mapdl.prep7()  # entering in preprocessor for geometry generation
+
+k0 = mapdl.k("", 0, 0, 0)
+k1 = mapdl.k("", 1, 0, 0)
+k2 = mapdl.k("", 0, 1, 0)
+a0 = mapdl.a(k0, k1, k2)
+mapdl.aplot(show_lines=True, line_width=5, show_bounds=True, cpos="xy")
+
+##################################
+# Create a simple cube volume.
+#
+mapdl.clear()  # let's clear first.
+mapdl.prep7()  # entering in preprocessor
+
+k0 = mapdl.k("", 0, 0, 0)  # defining a keypoint at (0,0,0) location
+k1 = mapdl.k("", 1, 0, 0)  # defining a keypoint at (1,0,0) location
+k2 = mapdl.k("", 1, 1, 0)  # etc
+k3 = mapdl.k("", 0, 1, 0)
+k4 = mapdl.k("", 0, 0, 1)
+k5 = mapdl.k("", 1, 0, 1)
+k6 = mapdl.k("", 1, 1, 1)
+k7 = mapdl.k("", 0, 1, 1)
+
+# defining volume according the kps
+v0 = mapdl.v(k0, k1, k2, k3, k4, k5, k6, k7)
+
+mapdl.vplot(show_lines=True)
+
+###################################
+# Material definition
+# -------------------
+#
+# You can define materials using the following commands:
+
+# Define a material (nominal steel in SI)
+mapdl.mp("EX", 1, 210e9)  # Elastic moduli in Pa (kg/(m*s**2))
+mapdl.mp("DENS", 1, 7800)  # Density in kg/m3
+mapdl.mp("NUXY", 1, 0.3)  # Poisson's Ratio
+
+###################################
+# At any point you can use help to get information about each
+# function
+help(mapdl.mp)
+
+#####################################
+# Or you can check the online help at
+# `mapdl.docs.pyansys.com <https://mapdl.docs.pyansys.com/mapdl_commands/prep7/_autosummary/ansys.mapdl.core.Mapdl.mp.html?highlight=mp#ansys.mapdl.core.Mapdl.mp>`_
+
+########################################
+# Element definition
+# ------------------
+#
+# Since MAPDL is a finite element solver,
+# the FE element needs to be defined.
```

Comment:
the type of Finite Element?

Reply:
Good point.

It should say the type of element.